tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -913,7 +913,7 @@ public boolean consumed(final MessageExt msg,
         final String group) throws RemotingException, MQClientException, InterruptedException,
         MQBrokerException {
 
-        ConsumeStats cstats = this.examineConsumeStats(group);
+        ConsumeStats cstats = this.examineConsumeStats(group,msg.getTopic());
 
         ClusterInfo ci = this.examineBrokerClusterInfo();
 
```

Comment:
The following codes are used to check topic . Maybe you can simplify it.

https://github.com/apache/rocketmq/blob/e7ee1186b5f1b9a46da67e3d71cc7e01c27600bd/tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java#L921

Reply:
got it, but i still confuse. because examineConsumeStats method send group&topic to broker, the broker can filter the topic's message.  why admin client retrieve all topic of the group and filter the topic's message?